js/scripts.js

```diff
@@ -0,0 +1,9 @@
+﻿var a = 0,
+	h = 0,
+	triangleArea;
+
+var a = prompt('Enter triangle base');
```

Comment:
remove `var` word. You need to add var when you want to declare a variable, when you change value of exisiting variable we don't declare another time same variable